test/integration/policy_set_test.go

```diff
@@ -0,0 +1,247 @@
+// Copyright (c) 2022 Red Hat, Inc.
+// Copyright Contributors to the Open Cluster Management project
+
+package integration
+
+import (
+	"context"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	policiesv1 "github.com/stolostron/governance-policy-propagator/api/v1"
+	"github.com/stolostron/governance-policy-propagator/controllers/common"
+	"github.com/stolostron/governance-policy-propagator/test/utils"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+
+	testcommon "github.com/stolostron/governance-policy-framework/test/common"
+)
+
+const (
+	testPolicyName          string = "test-policy"
+	testPolicySetName       string = "test-policyset"
+	testPolicyYaml          string = "../resources/policy_set/test-policy.yaml"
+	testPolicySetPatchYaml  string = "../resources/policy_set/patch-policy-set.yaml"
+	testedDisablePolicyYaml string = "../resources/policy_set/disable-policy.yaml"
+)
+
+var _ = Describe("GRC: [P1][Sev1][policy-grc] Test policy set", func() {
+	Describe("Create policy, policyset, and placement in ns:"+userNamespace, func() {
+		It("Should create and process policy and policyset", func() {
+			By("Creating " + testPolicyYaml)
+			output, err := testcommon.OcHub("apply",
```

Comment:
`testcommon.OcHub` and `utils.KubectlWithOutput` are both used in your test case. Suggest to use one and make it consistent in your test case.